src/app/core/services/issue.service.ts

```diff
@@ -25,6 +25,8 @@ import { TesterResponse } from '../models/tester-response.model';
 import { IssueComments, IssueComment } from '../models/comment.model';
 import { IssueDispute } from '../models/issue-dispute.model';
 import { UserRole } from '../../core/models/user.model';
+import {BaseIssue} from '../models/base-issue.model';
```

Comment:
Can include the space between the braces like the above import statements to standardise?
Err same for the one below 😅